charts/pega/README.md

```diff
@@ -435,10 +435,11 @@ Parameter           | Description    | Default value
 ---                 | ---       | ---
 `hpa.minReplicas`   | Minimum number of replicas that HPA can scale-down | `1` 
 `hpa.maxReplicas`   | Maximum number of replicas that HPA can scale-up  | `5`
-`hpa.targetAverageCPUUtilization` | Threshold value for scaling based on initial CPU request utilization (The default value is `70` which corresponds to 70% of 2) | `70`
-`hpa.targetAverageMemoryUtilization` | Threshold value for scaling based on initial memory utilization (The default value is `85` which corresponds to 85% of 6Gi ) | `85`
+`hpa.targetAverageCPUValue` | Threshold value for scaling based on absolute CPU usage | `2.8`
```

Comment:
Can you please add what 2.8 represents? A context would be useful to lead clients to make decisions and not guess.

Reply:
How about now?  Updated to include a basic description.

Reply:
can you cite an article that tells me (or anyone that is in a position to want to begin scaling), like a test engineer, how I can see 2.8? Previously they explained that 85 meant 85% of 6Gi. Can you give a hint what 2.8 is? I'm 97.2% sure that you didn't just make 2.8 up. ;-)

Reply:
You should check the most recent version of the file, the thing shown in this conversation doesn't update.